sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -31,66 +32,69 @@
-        "operationId": "Configuration_CreateOrUpdate",
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "name": "id",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "configuration",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/Configuration"
-            }
-          },
-          {
-            "name": "If-Match",
-            "in": "header",
-            "required": false,
-            "type": "string"
-          },
-          {
-            "$ref": "#/parameters/api-version"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Returns the updated Configuration object",
-            "schema": {
-              "$ref": "#/definitions/Configuration"
-            }
-          },
-          "201": {
-            "description": "Returns the created Configuration object",
-            "schema": {
-              "$ref": "#/definitions/Configuration"
+        "put": {
+            "summary": "Create or update the configuration for Iot Hub device or module.",
+            "description": "Create or update the configuration for Iot Hub device or module. ETag must not be specified for the create operation. ETag must be specified for the update operation. Configuration Id and Content cannot be updated.",
+            "operationId": "Configuration_CreateOrUpdate",
+            "consumes": [
+                "application/json"
+            ],
+            "produces": [
+                "application/json"
+            ],
+            "parameters": [
+                {
+                    "name": "id",
+                    "description": "The configuration Id.",
+                    "in": "path",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "configuration",
+                    "description": "The configuration to be created or updated.",
+                    "in": "body",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/definitions/Configuration"
+                    }
+                },
+                {
+                    "name": "If-Match",
+                    "description": "Header to make the request conditional. The operation is performed only if the ETag matches the value in the server. To force an unconditional update, set If-Match to the wildcard character (*).",
```

Comment:
When describing the Etag, there is an RFC that is worth referencing. I made some mentions towards it in the registry APIs that I recommend you also use

Reply:
Lets sync offline on this. I think I agree with Abhipsa that we should have the same description for this in all APIs.